nextcloudappstore/core/templates/app/detail.html

```diff
@@ -13,7 +13,7 @@
     <script defer="defer" src="{% static 'assets/js/ui/app-detail.js' %}"></script>
     <meta name="nextcloudappstore-app-description-url" content="{% url 'app-description' object.id %}">
     <meta name="nextcloudappstore-app-ratings-url" content="{% url 'app-ratings' object.id %}">
-    <meta name="language-code" content="{{ request.LANGUAGE_CODE }}">
+    <meta name="language-code" content="{{ language_code }}">
```

Comment:
Should probably stay the same (since its the global language code). If this is actually the global language code we should bind it to the request using a middleware.